frontend/packages/topology/src/components/graph-view/TopologyShortcuts.tsx

```diff
@@ -1,23 +1,30 @@
 import * as React from 'react';
 import { TFunction } from 'i18next';
 import { Shortcut, ShortcutTable } from '@console/shared';
+import { TopologyViewType } from '../../topology-types';
 
 export type Options = {
   supportedFileTypes: string[];
+  isEmptyModel: boolean;
+  viewType: TopologyViewType;
 };
 export const getTopologyShortcuts = (t: TFunction, options: Options): React.ReactElement => (
```

Comment:
Since this is rendering components conditionally, we should have a unit test for it.

Reply:
added TopologyShortcuts.spec.tsx